### ISSUE TITLE
Add possibility to kill stalled process, when process is started from web-UI

### ISSUE THREAD

**Issue by MW3000** [DSpace/dspace-angular#2233]
**Describe the bug**
One of our users started a small (10 items) metadata-import process that run into a bug presumably and never finished. It was still running after 24h. If started via CLI, we could just `ctrl-c` the process, but started via web-UI we found no possibility to kill the process.

**Expected behavior**
There should be a way to kill a stalled process, when the process is started from web-UI. Preferable, killing such a process would be possible both via web-UI and via CLI.

**Comment by kshepherd** [comment]
i believe this is possible through the main "processes" screen but not the page for the process itself? taht said, i have not determined whether this actually kills any background process or just removes it from being tracked by dspace...

**Comment by tdonohue** [comment]
@kshepherd : I don't believe it is currently possible to stop a process from the Admin UI.  From the "processes" Admin UI page, I think there's just a delete button which deletes the Process & it's related files from the processes table (in the DB).

But I don't believe "delete" does anything to "kill" or stop a process that is still in the "running" state.  

Flagging this as "medium priority" for now.  Volunteers welcome
